lib/handlers.js

```diff
@@ -53,6 +53,27 @@ module.exports = (self) => {
         self.apos.doc.addContextOperation(exportOperation);
         self.apos.doc.addContextOperation(importOperation);
       }
+    },
+    'apostrophe:destroy': {
+      async clearTimers() {
+        const ids = Object.keys(self.timeoutIds);
+        if (!ids.length) {
+          return;
+        }
+        self.apos.util.debug(`Clearing ${ids.length} timer(s)`);
+        for (const key of ids) {
+          const entry = self.timeoutIds[key];
+          delete self.timeoutIds[key];
+          if (!entry) {
```

Comment:
Can this really happen? It's OK though

Reply:
Yeah, I was just extra cautious to not break stuff while we are destroying. I'd say it's not possible, but not harmful either.